scripts/spell_check/check_spelling.sh

```diff
@@ -41,6 +41,9 @@ shift $(expr $OPTIND - 1)
 if [ ! $# -eq 0 ]; then
   EXCLUDE=$(cat $AGIGNORE | sed -e 's/\s*#.*$//' -e '/^\s*$/d' | tr '\n' '|' | sed -e 's/|$//')
   INPUTFILES=$(echo $@ | tr -s '[[:blank:]]' '\n' | egrep -iv "$EXCLUDE" | tr '\n' ' ' )
+  if [ "$INPUTFILES" == "" ]; then
+    exit 0
+  fi
```

Comment:
does it still work on all files when calling without argument?
I don't understand the aim of this check.